cmd/openshift-tests/e2e.go

```diff
@@ -43,6 +43,15 @@ var staticSuites = []*ginkgo.TestSuite{
 			return strings.Contains(name, "[Suite:openshift/conformance/serial")
 		},
 	},
+	{
+		Name: "openshift/multitenant/conformance",
+		Description: templates.LongDesc(`
+		Only the portion of the openshift/multitenant test suite.
```

Comment:
"Only the portion of the openshift/conformance test suite that applies to the openshift-sdn multitenant plugin."